DESCRIPTION

```diff
@@ -12,7 +12,10 @@ Description: The package provides convenient access to StatCan census data via C
 License: MIT
 Encoding: UTF-8
 LazyData: true
-Depends: dplyr (>= 0.7.2),  httr (>= 1.2.1), jsonlite (>= 1.5), digest (>= 0.6.12)
+Imports: digest (>= 0.1),
+    dplyr (>= 0.7.2),
```

Comment:
Something to monitor here is the ongoing compatibility/stability between `sf` and `dplyr`. This thread highlights some issues (https://github.com/r-spatial/sf/issues/42) and both Jens and I have run into some errors at times. I think what we have here, with requiring the latest versions is fine, however.